styx-common/src/main/java/com/spotify/styx/testdata/TestData.java

```diff
@@ -140,6 +143,37 @@
           .retryCondition("#exitCode == 1 && (#tries < 3 || #consecutiveFailures < 4) && #triggerType == \"natural\"")
           .build();
 
+  public static final WorkflowConfiguration DOCKER_EXEC_WORKFLOW_CONFIGURATION =
+      WorkflowConfiguration.builder()
+          .id("styx.TestEndpoint")
+          .commitSha(VALID_SHA)
+          .schedule(DAYS)
+          .serviceAccount("foo@bar.baz.quux")
+          .dockerExecConf(new DockerExecConfBuilder()
+              .dockerImage("busybox")
+              .dockerArgs(List.of("x", "y"))
+              .dockerTerminationLogging(true)
+              .build())
+          .build();
+
+  public static final WorkflowConfiguration FLYTE_WORKFLOW_CONFIGURATION =
+      WorkflowConfiguration.builder()
+          .id("styx.TestEndpoint")
+          .commitSha(VALID_SHA)
+          .schedule(DAYS)
+          .serviceAccount("foo@bar.baz.quux")
+          .flyteExecConf(new FlyteExecConfBuilder()
+              .referenceId(new FlyteIdentifierBuilder()
```

Comment:
Let's make the Builder usage consistent with the WorkflowConfiguration.builder()